src/pluginManager.ts

```diff
@@ -24,6 +24,7 @@ export interface PackageJSON { // incomplete type for package.json (just stuff w
   keywords?: string[];
 
   main?: string;
+  exports?: string | Record<string, string | Record<string, string>>
```

Comment:
Would be great if you could include the link to the docs https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_package_entry_points into the docs comment for the field.

Reply:
Done!